src/main/java/com/example/shorturl/controller/api/ShortenUrlControllerV2.java

```diff
@@ -33,16 +42,58 @@ public ResponseEntity<UserUrlResponse> shortUrl(@RequestBody UserUrlRequest user
         if(Objects.isNull(userUrlRequest) || StringUtils.isEmpty(userUrlRequest.getUrl()))
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Url request is missing or empty");
 
-        UrlEntity url = shortenUrlService.userShortUrl(userUrlRequest);
+        if(shortenUrlService.existsByAlias(userUrlRequest.getAlias()))
```

Comment:
If user wants to use the same alias, we should not return any error, let's return the same alias

Reply:
What if the user has 2 urls with the same alias? I think it will be a problem in this case. But if the same alias belongs to diferrent users ther no problem

Reply:
Yes, that's what I am talking about. If user wants to use the same alias, don't create it or throw exception, just return that alias from DB

Reply:
Got it

Reply:
Fixed